### PR TITLE
allow use of addContext from mochawesome or standalone

### PR DESCRIPTION
**Pull request by newpablo**

### CODE REVIEW

File: src/elastic-search.js [hunk 1]
```diff
@@ -53,7 +69,7 @@ module.exports = function sendTestResults(testResultsLog, done) {
             duration: failedTestData[arrayLength].duration / 1000,
             err: failedTestData[arrayLength].err,
             uuid: failedTestData[arrayLength].title.match(uuidRegex)
-        }, extraParams)
+        }, extraParams,contextValues)
```
Comment (screighton) [inline src/elastic-search.js]: add space between variables

File: src/elastic-search.js [hunk 2]
```diff
@@ -27,22 +27,38 @@ module.exports = function sendTestResults(testResultsLog, done) {
...
         "content":
         Object.assign({ status: "passed",
-          title: passedTestData[arrayLength].title,
+          title: passedTestData[arrayLength].title,             
           fullTitle: passedTestData[arrayLength].fullTitle,
           duration: passedTestData[arrayLength].duration / 1000,
           err: passedTestData[arrayLength].err,
           uuid: passedTestData[arrayLength].title.match(uuidRegex)
-        }, extraParams)
+        }, extraParams,contextValues)
```
Comment (screighton) [inline src/elastic-search.js]: add space between variables

File: src/elastic-search.js [hunk 3]
```diff
@@ -27,22 +27,38 @@ module.exports = function sendTestResults(testResultsLog, done) {
...
+        }
+       }
+
       resultsArray.push({
         "application_name": repoConfig.applicationName,
         "et": timestamp,
         "content":
         Object.assign({ status: "passed",
-          title: passedTestData[arrayLength].title,
+          title: passedTestData[arrayLength].title,
```
Comment (screighton) [inline src/elastic-search.js]: extra tab/spacing at end can be removed

File: src/elastic-search.js [hunk 4]
```diff
@@ -27,22 +27,38 @@ module.exports = function sendTestResults(testResultsLog, done) {
     var uuidRegex = /\[(uuid:.*?)\]/g;
     var timestamp = new Date().getTime();
     for (var arrayLength = 0; arrayLength < passedTestDataSize; arrayLength++ ) {
+
+      let contextValues; 
+      if(passedTestData[arrayLength].context !== undefined){
```
Comment (screighton) [inline src/elastic-search.js]: code formatting doesn't match... on line 29 there's a space between for and begin paren, also a space between end paren and bracket... should keep this consistent throughout the code

File: src/elastic-search.js (hunk 1 above)
Comment (newpablo) [reply src/elastic-search.js]: fixed in next checkin

File: src/elastic-search.js (hunk 2 above)
Comment (newpablo) [reply src/elastic-search.js]: fixed in next checkin

File: src/elastic-search.js (hunk 3 above)
Comment (newpablo) [reply src/elastic-search.js]: fixed in next checkin

File: src/elastic-search.js (hunk 4 above)
Comment (newpablo) [reply src/elastic-search.js]: fixed in next checkin

**Pull request merged**